src/test/java/org/pesmypetcare/webservice/dao/communitymanager/ForumDaoTest.java

```diff
@@ -410,7 +458,8 @@ public void addUserToLikedByOfMessage() throws DatabaseAccessException, Document
                     verify(collectionAdapter)
                         .getDocumentsWhereEqualTo(eq(Path.ofCollection(Collections.messages, groupId, forumId)),
                             eq("creator"), eq(username), eq(publicationDate), eq(date));
-                    verify(batch).update(same(documentReference), eq("likedBy"), eq(FieldValue.arrayUnion(username)));
+                    verify(batch).update(same(documentReference), eq("likedBy"),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "likedBy" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5633182)